### ISSUE TITLE
Crash upon opening

### ISSUE THREAD

**Issue by bloodconfetti** [Glitchfiend/BiomesOPlenty#1444]
## Bug Report
Crash</br>

### How can the crash be reproduced?

I an not sure how to reproduce the crash. It was working by itself, and then with the mod absent by design, and animal net - but then I added attained drops, appleskin, and placebo, and then it crashed to a page stating 
![biomecrash](https://user-images.githubusercontent.com/53692357/62753149-85a22e00-ba27-11e9-9fc2-d736c8307a4d.png)

</br>

### Crash Report and Logs
[08Aug2019 21:49:56.130] [main/INFO] [cpw.mods.modlauncher.Launcher/MODLAUNCHER]: ModLauncher running: args [--username, willowswaying, --version, 1.14.4-forge-28.0.45, --gameDir, C:\Users\Jack\AppData\Roaming\.minecraft, --assetsDir, C:\Users\Jack\AppData\Roaming\.minecraft\assets, --assetIndex, 1.14, --uuid, 8ced1a1ccf3149909785f4dfadba1c39, --accessToken, ????????, --userType, mojang, --versionType, release, --width, 1600, --height, 900, --launchTarget, fmlclient, --fml.forgeVersion, 28.0.45, --fml.mcVersion, 1.14.4, --fml.forgeGroup, net.minecraftforge, --fml.mcpVersion, 20190719.225934]
[08Aug2019 21:49:56.133] [main/INFO] [cpw.mods.modlauncher.Launcher/MODLAUNCHER]: ModLauncher 3.2.0+60+b86c1d4 starting: java version 1.8.0_51 by Oracle Corporation
[08Aug2019 21:49:56.340] [main/INFO] [net.minecraftforge.fml.loading.FixSSL/CORE]: Added Lets Encrypt root certificates as additional trust
[08Aug2019 21:49:56.994] [main/INFO] [cpw.mods.modlauncher.LaunchServiceHandler/MODLAUNCHER]: Launching target 'fmlclient' with arguments [--version, 1.14.4-forge-28.0.45, --gameDir, C:\Users\Jack\AppData\Roaming\.minecraft, --assetsDir, C:\Users\Jack\AppData\Roaming\.minecraft\assets, --username, willowswaying, --assetIndex, 1.14, --uuid, 8ced1a1ccf3149909785f4dfadba1c39, --accessToken, ????????, --userType, mojang, --versionType, release, --width, 1600, --height, 900]
[08Aug2019 21:49:58.388] [Client thread/INFO] [net.minecraft.client.Minecraft/]: Setting user: willowswaying
[08Aug2019 21:50:06.121] [Client thread/INFO] [net.minecraft.client.Minecraft/]: LWJGL Version: 3.2.2 build 10
[08Aug2019 21:50:07.756] [Client thread/WARN] [net.minecraft.resources.ResourcePackInfo/]: Couldn't get pack info for: net.minecraft.resources.ResourcePackFileNotFoundException: 'pack.mcmeta' in ResourcePack 'C:\Users\Jack\AppData\Roaming\.minecraft\resourcepacks\Revival-master.zip'
[08Aug2019 21:50:08.175] [modloading-worker-6/INFO] [net.minecraftforge.common.ForgeMod/FORGEMOD]: Forge mod loading, version 28.0.45, for MC 1.14.4 with MCP 20190719.225934
[08Aug2019 21:50:08.177] [modloading-worker-6/INFO] [net.minecraftforge.common.MinecraftForge/FORGE]: MinecraftForge v28.0.45 Initialized
[08Aug2019 21:50:08.351] [modloading-worker-5/WARN] [net.minecraft.entity.EntityType/]: No data fixer registered for entity biomesoplenty:boat_bop
[08Aug2019 21:50:08.347] [Client thread/ERROR] [net.minecraftforge.fml.ModList/LOADING]: Encountered an exception during parallel processing
java.util.concurrent.ExecutionException: java.lang.NoClassDefFoundError
	at java.util.concurrent.ForkJoinTask.get(ForkJoinTask.java:1006) ~[?:1.8.0_51]
	at net.minecraftforge.fml.ModList.dispatchParallelEvent(ModList.java:137) [?:?]
	at net.minecraftforge.fml.ModList.lambda$static$2(ModList.java:97) [?:?]
	at net.minecraftforge.fml.ModList$$Lambda$1970/1956658133.dispatchEvent(Unknown Source) [?:?]
	at net.minecraftforge.fml.LifecycleEventProvider.dispatch(LifecycleEventProvider.java:71) [?:?]
	at net.minecraftforge.fml.ModLoader.dispatchAndHandleError(ModLoader.java:194) [?:?]
	at net.minecraftforge.fml.ModLoader.gatherAndInitializeMods(ModLoader.java:180) [?:?]
	at net.minecraftforge.fml.client.ClientModLoader.lambda$begin$2(ClientModLoader.java:79) [?:?]
	at net.minecraftforge.fml.client.ClientModLoader$$Lambda$1952/1668613848.run(Unknown Source) [?:?]
	at net.minecraftforge.fml.client.ClientModLoader.lambda$createRunnableWithCatch$5(ClientModLoader.java:95) [?:?]
	at net.minecraftforge.fml.client.ClientModLoader$$Lambda$1953/1549856918.run(Unknown Source) [?:?]
	at net.minecraftforge.fml.client.ClientModLoader.begin(ClientModLoader.java:79) [?:?]
	at net.minecraft.client.Minecraft.func_71384_a(Minecraft.java:453) [?:?]
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:365) [?:?]
	at net.minecraft.client.main.Main.main(SourceFile:155) [?:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_51]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_51]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_51]
	at java.lang.reflect.Method.invoke(Method.java:497) ~[?:1.8.0_51]
	at net.minecraftforge.fml.loading.FMLClientLaunchProvider.lambda$launchService$0(FMLClientLaunchProvider.java:56) [forge-1.14.4-28.0.45.jar:28.0]
	at net.minecraftforge.fml.loading.FMLClientLaunchProvider$$Lambda$342/1397333381.call(Unknown Source) [forge-1.14.4-28.0.45.jar:28.0]
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:37) [modlauncher-3.2.0.jar:?]
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:50) [modlauncher-3.2.0.jar:?]
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:68) [modlauncher-3.2.0.jar:?]
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:80) [modlauncher-3.2.0.jar:?]
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:65) [modlauncher-3.2.0.jar:?]
Caused by: java.lang.NoClassDefFoundError
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:1.8.0_51]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[?:1.8.0_51]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:1.8.0_51]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[?:1.8.0_51]
	at java.util.concurrent.ForkJoinTask.getThrowableException(ForkJoinTask.java:598) ~[?:1.8.0_51]
	at java.util.concurrent.ForkJoinTask.get(ForkJoinTask.java:1005) ~[?:1.8.0_51]
	... 25 more
Caused by: java.lang.NoClassDefFoundError
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:1.8.0_51]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[?:1.8.0_51]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:1.8.0_51]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[?:1.8.0_51]
	at java.util.concurrent.ForkJoinTask.getThrowableException(ForkJoinTask.java:598) ~[?:1.8.0_51]
	at java.util.concurrent.ForkJoinTask.reportException(ForkJoinTask.java:677) ~[?:1.8.0_51]
	at java.util.concurrent.ForkJoinTask.invoke(ForkJoinTask.java:735) ~[?:1.8.0_51]
	at java.util.stream.ForEachOps$ForEachOp.evaluateParallel(ForEachOps.java:160) ~[?:1.8.0_51]
	at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateParallel(ForEachOps.java:174) ~[?:1.8.0_51]
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:233) ~[?:1.8.0_51]
	at java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:418) ~[?:1.8.0_51]
	at java.util.stream.ReferencePipeline$Head.forEach(ReferencePipeline.java:583) ~[?:1.8.0_51]
	at net.minecraftforge.fml.ModList.lambda$dispatchParallelEvent$11(ModList.java:133) ~[?:?]
	at net.minecraftforge.fml.ModList$$Lambda$2063/40583144.run(Unknown Source) ~[?:?]
	at java.util.concurrent.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1386) ~[?:1.8.0_51]
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289) ~[?:1.8.0_51]
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056) ~[?:1.8.0_51]
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1689) ~[?:1.8.0_51]
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157) ~[?:1.8.0_51]
Caused by: java.lang.NoClassDefFoundError: net/minecraftforge/fml/common/gameevent/PlayerEvent$ItemCraftedEvent
	at java.lang.Class.getDeclaredMethods0(Native Method) ~[?:1.8.0_51]
	at java.lang.Class.privateGetDeclaredMethods(Class.java:2701) ~[?:1.8.0_51]
	at java.lang.Class.privateGetPublicMethods(Class.java:2902) ~[?:1.8.0_51]
	at java.lang.Class.getMethods(Class.java:1615) ~[?:1.8.0_51]
	at net.minecraftforge.eventbus.EventBus.registerClass(EventBus.java:69) ~[eventbus-0.10.6-service.jar:?]
	at net.minecraftforge.eventbus.EventBus.register(EventBus.java:112) ~[eventbus-0.10.6-service.jar:?]
	at net.minecraftforge.fml.AutomaticEventSubscriber.lambda$inject$3(AutomaticEventSubscriber.java:70) ~[?:?]
	at net.minecraftforge.fml.AutomaticEventSubscriber$$Lambda$2093/306196724.accept(Unknown Source) ~[?:?]
	at java.util.ArrayList.forEach(ArrayList.java:1249) ~[?:1.8.0_51]
	at net.minecraftforge.fml.AutomaticEventSubscriber.inject(AutomaticEventSubscriber.java:57) ~[?:?]
	at net.minecraftforge.fml.javafmlmod.FMLModContainer.constructMod(FMLModContainer.java:140) ~[?:28.0]
	at net.minecraftforge.fml.javafmlmod.FMLModContainer$$Lambda$1998/1980780021.accept(Unknown Source) ~[?:?]
	at java.util.function.Consumer.lambda$andThen$14(Consumer.java:65) ~[?:1.8.0_51]
	at java.util.function.Consumer$$Lambda$1997/170974144.accept(Unknown Source) ~[?:?]
	at java.util.function.Consumer.lambda$andThen$14(Consumer.java:65) ~[?:1.8.0_51]
	at java.util.function.Consumer$$Lambda$1997/170974144.accept(Unknown Source) ~[?:?]
	at net.minecraftforge.fml.ModContainer.transitionState(ModContainer.java:112) ~[?:?]
	at net.minecraftforge.fml.ModList.lambda$null$10(ModList.java:133) ~[?:?]
	at net.minecraftforge.fml.ModList$$Lambda$2065/1707302112.accept(Unknown Source) ~[?:?]
	at java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:184) ~[?:1.8.0_51]
	at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1374) ~[?:1.8.0_51]
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:512) ~[?:1.8.0_51]
	at java.util.stream.ForEachOps$ForEachTask.compute(ForEachOps.java:291) ~[?:1.8.0_51]
	at java.util.concurrent.CountedCompleter.exec(CountedCompleter.java:731) ~[?:1.8.0_51]
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289) ~[?:1.8.0_51]
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056) ~[?:1.8.0_51]
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1689) ~[?:1.8.0_51]
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157) ~[?:1.8.0_51]
Caused by: java.lang.ClassNotFoundException: net.minecraftforge.fml.common.gameevent.PlayerEvent$ItemCraftedEvent
	at java.lang.ClassLoader.findClass(ClassLoader.java:530) ~[?:1.8.0_51]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424) ~[?:1.8.0_51]
	at cpw.mods.modlauncher.TransformingClassLoader.loadClass(TransformingClassLoader.java:102) ~[modlauncher-3.2.0.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357) ~[?:1.8.0_51]
	at java.lang.Class.getDeclaredMethods0(Native Method) ~[?:1.8.0_51]
	at java.lang.Class.privateGetDeclaredMethods(Class.java:2701) ~[?:1.8.0_51]
	at java.lang.Class.privateGetPublicMethods(Class.java:2902) ~[?:1.8.0_51]
	at java.lang.Class.getMethods(Class.java:1615) ~[?:1.8.0_51]
	at net.minecraftforge.eventbus.EventBus.registerClass(EventBus.java:69) ~[eventbus-0.10.6-service.jar:?]
	at net.minecraftforge.eventbus.EventBus.register(EventBus.java:112) ~[eventbus-0.10.6-service.jar:?]
	at net.minecraftforge.fml.AutomaticEventSubscriber.lambda$inject$3(AutomaticEventSubscriber.java:70) ~[?:?]
	at net.minecraftforge.fml.AutomaticEventSubscriber$$Lambda$2093/306196724.accept(Unknown Source) ~[?:?]
	at java.util.ArrayList.forEach(ArrayList.java:1249) ~[?:1.8.0_51]
	at net.minecraftforge.fml.AutomaticEventSubscriber.inject(AutomaticEventSubscriber.java:57) ~[?:?]
	at net.minecraftforge.fml.javafmlmod.FMLModContainer.constructMod(FMLModContainer.java:140) ~[?:28.0]
	at net.minecraftforge.fml.javafmlmod.FMLModContainer$$Lambda$1998/1980780021.accept(Unknown Source) ~[?:?]
	at java.util.function.Consumer.lambda$andThen$14(Consumer.java:65) ~[?:1.8.0_51]
	at java.util.function.Consumer$$Lambda$1997/170974144.accept(Unknown Source) ~[?:?]
	at java.util.function.Consumer.lambda$andThen$14(Consumer.java:65) ~[?:1.8.0_51]
	at java.util.function.Consumer$$Lambda$1997/170974144.accept(Unknown Source) ~[?:?]
	at net.minecraftforge.fml.ModContainer.transitionState(ModContainer.java:112) ~[?:?]
	at net.minecraftforge.fml.ModList.lambda$null$10(ModList.java:133) ~[?:?]
	at net.minecraftforge.fml.ModList$$Lambda$2065/1707302112.accept(Unknown Source) ~[?:?]
	at java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:184) ~[?:1.8.0_51]
	at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1374) ~[?:1.8.0_51]
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:512) ~[?:1.8.0_51]
	at java.util.stream.ForEachOps$ForEachTask.compute(ForEachOps.java:291) ~[?:1.8.0_51]
	at java.util.concurrent.CountedCompleter.exec(CountedCompleter.java:731) ~[?:1.8.0_51]
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289) ~[?:1.8.0_51]
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056) ~[?:1.8.0_51]
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1689) ~[?:1.8.0_51]
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157) ~[?:1.8.0_51]
[08Aug2019 21:50:08.642] [modloading-worker-5/ERROR] [net.minecraftforge.fml.javafmlmod.FMLModContainer/LOADING]: Failed to create mod instance. ModID: biomesoplenty, class biomesoplenty.core.BiomesOPlenty
java.lang.IllegalStateException: The object biomesoplenty.common.biome.overworld.MangroveBiome@5e392786 (name biomesoplenty:mangrove) is being added too late.
	at net.minecraftforge.registries.ForgeRegistry.add(ForgeRegistry.java:348) ~[?:?]
	at net.minecraftforge.registries.ForgeRegistry.add(ForgeRegistry.java:308) ~[?:?]
	at net.minecraftforge.registries.ForgeRegistry.register(ForgeRegistry.java:133) ~[?:?]
	at biomesoplenty.init.ModBiomes.registerBiome(ModBiomes.java:295) ~[?:?]
	at biomesoplenty.init.ModBiomes.registerBiomes(ModBiomes.java:112) ~[?:?]
	at biomesoplenty.init.ModBiomes.init(ModBiomes.java:105) ~[?:?]
	at biomesoplenty.core.BiomesOPlenty.<init>(BiomesOPlenty.java:49) ~[?:?]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:1.8.0_51]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[?:1.8.0_51]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:1.8.0_51]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:422) ~[?:1.8.0_51]
	at java.lang.Class.newInstance(Class.java:442) ~[?:1.8.0_51]
	at net.minecraftforge.fml.javafmlmod.FMLModContainer.constructMod(FMLModContainer.java:131) [?:28.0]
	at net.minecraftforge.fml.javafmlmod.FMLModContainer$$Lambda$1998/1980780021.accept(Unknown Source) [?:28.0]
	at java.util.function.Consumer.lambda$andThen$14(Consumer.java:65) [?:1.8.0_51]
	at java.util.function.Consumer$$Lambda$1997/170974144.accept(Unknown Source) [?:1.8.0_51]
	at java.util.function.Consumer.lambda$andThen$14(Consumer.java:65) [?:1.8.0_51]
	at java.util.function.Consumer$$Lambda$1997/170974144.accept(Unknown Source) [?:1.8.0_51]
	at net.minecraftforge.fml.ModContainer.transitionState(ModContainer.java:112) [?:?]
	at net.minecraftforge.fml.ModList.lambda$null$10(ModList.java:133) [?:?]
	at net.minecraftforge.fml.ModList$$Lambda$2065/1707302112.accept(Unknown Source) [?:?]
	at java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:184) [?:1.8.0_51]
	at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1374) [?:1.8.0_51]
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:512) [?:1.8.0_51]
	at java.util.stream.ForEachOps$ForEachTask.compute(ForEachOps.java:291) [?:1.8.0_51]
	at java.util.concurrent.CountedCompleter.exec(CountedCompleter.java:731) [?:1.8.0_51]
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289) [?:1.8.0_51]
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056) [?:1.8.0_51]
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1689) [?:1.8.0_51]
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157) [?:1.8.0_51]
[08Aug2019 21:50:08.648] [Client thread/WARN] [net.minecraft.resources.ResourcePackInfo/]: Couldn't get pack info for: net.minecraft.resources.ResourcePackFileNotFoundException: 'pack.mcmeta' in ResourcePack 'C:\Users\Jack\AppData\Roaming\.minecraft\resourcepacks\Revival-master.zip'
[08Aug2019 21:50:09.215] [Client thread/INFO] [com.mojang.text2speech.NarratorWindows/]: Narrator library for x64 successfully loaded
[08Aug2019 21:50:09.314] [Thread-1/FATAL] [net.minecraftforge.common.ForgeConfig/CORE]: Forge config just got changed on the file system!
[08Aug2019 21:50:09.373] [Server-Worker-4/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Configuration file C:\Users\Jack\AppData\Roaming\.minecraft\config\animalnet-common.toml is not correct. Correcting
[08Aug2019 21:50:09.374] [Server-Worker-4/WARN] [net.minecraftforge.common.ForgeConfigSpec/CORE]: Incorrect key General.excluded_entities was corrected from [minecraft:wither, minecraft:ender_dragon] to [minecraft:wither, minecraft:ender_dragon]
[08Aug2019 21:50:09.389] [Server-Worker-4/INFO] [attained_drops/]: Loaded config file!
[08Aug2019 21:50:09.396] [Server-Worker-4/ERROR] [net.minecraftforge.fml.ModLoader/LOADING]: Skipping lifecycle event SETUP, 1 errors found.
[08Aug2019 21:50:09.396] [Server-Worker-4/FATAL] [net.minecraftforge.fml.ModLoader/LOADING]: Failed to complete lifecycle event SETUP, 1 errors found
[08Aug2019 21:50:09.396] [Server-Worker-4/FATAL] [net.minecraftforge.eventbus.EventBus/EVENTBUS]: EventBus 0 shutting down - future events will not be posted.
java.lang.Exception: stacktrace
	at net.minecraftforge.eventbus.EventBus.shutdown(EventBus.java:278) [eventbus-0.10.6-service.jar:?]
	at net.minecraftforge.fml.client.ClientModLoader.lambda$createRunnableWithCatch$5(ClientModLoader.java:97) [?:?]
	at net.minecraftforge.fml.client.ClientModLoader$$Lambda$1953/1549856918.run(Unknown Source) [?:?]
	at net.minecraftforge.fml.client.ClientModLoader.startModLoading(ClientModLoader.java:105) [?:?]
	at net.minecraftforge.fml.client.ClientModLoader.lambda$onreload$3(ClientModLoader.java:87) [?:?]
	at net.minecraftforge.fml.client.ClientModLoader$$Lambda$2434/561566358.run(Unknown Source) [?:?]
	at net.minecraftforge.fml.client.ClientModLoader.lambda$createRunnableWithCatch$5(ClientModLoader.java:95) [?:?]
	at net.minecraftforge.fml.client.ClientModLoader$$Lambda$1953/1549856918.run(Unknown Source) [?:?]
	at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1618) [?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1610) [?:1.8.0_51]
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289) [?:1.8.0_51]
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056) [?:1.8.0_51]
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1689) [?:1.8.0_51]
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157) [?:1.8.0_51]
[08Aug2019 21:50:11.990] [Server-Worker-3/ERROR] [net.minecraft.client.renderer.texture.AtlasTexture/]: Using missing texture, unable to load minecraft:textures/block/light_bluewool.png : java.io.FileNotFoundException: minecraft:textures/block/light_bluewool.png
[08Aug2019 21:50:12.174] [Server-Worker-1/ERROR] [net.minecraftforge.fml.ModLoader/LOADING]: Skipping lifecycle event ENQUEUE_IMC, 1 errors found.
[08Aug2019 21:50:12.174] [Server-Worker-1/FATAL] [net.minecraftforge.fml.ModLoader/LOADING]: Failed to complete lifecycle event ENQUEUE_IMC, 1 errors found
[08Aug2019 21:50:12.175] [Server-Worker-1/FATAL] [net.minecraftforge.eventbus.EventBus/EVENTBUS]: EventBus 0 shutting down - future events will not be posted.
java.lang.Exception: stacktrace
	at net.minecraftforge.eventbus.EventBus.shutdown(EventBus.java:278) [eventbus-0.10.6-service.jar:?]
	at net.minecraftforge.fml.client.ClientModLoader.lambda$createRunnableWithCatch$5(ClientModLoader.java:97) [?:?]
	at net.minecraftforge.fml.client.ClientModLoader$$Lambda$1953/1549856918.run(Unknown Source) [?:?]
	at net.minecraftforge.fml.client.ClientModLoader.finishModLoading(ClientModLoader.java:118) [?:?]
	at net.minecraftforge.fml.client.ClientModLoader.lambda$onreload$4(ClientModLoader.java:89) [?:?]
	at net.minecraftforge.fml.client.ClientModLoader$$Lambda$2438/2026237784.run(Unknown Source) [?:?]
	at java.util.concurrent.CompletableFuture.uniRun(CompletableFuture.java:705) [?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$UniRun.tryFire(CompletableFuture.java:687) [?:1.8.0_51]
	at java.util.concurrent.CompletableFuture$Completion.exec(CompletableFuture.java:443) [?:1.8.0_51]
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289) [?:1.8.0_51]
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056) [?:1.8.0_51]
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1689) [?:1.8.0_51]
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157) [?:1.8.0_51]
[08Aug2019 21:50:12.274] [Client thread/INFO] [net.minecraft.client.audio.SoundSystem/]: OpenAL initialized.
[08Aug2019 21:50:12.275] [Client thread/INFO] [net.minecraft.client.audio.SoundEngine/SOUNDS]: Sound engine started
[08Aug2019 21:50:12.396] [Client thread/INFO] [net.minecraft.client.renderer.texture.AtlasTexture/]: Created: 1024x512 textures-atlas
[08Aug2019 21:50:13.003] [Client thread/INFO] [net.minecraft.client.renderer.texture.AtlasTexture/]: Created: 256x256 textures/particle-atlas
[08Aug2019 21:50:13.011] [Client thread/INFO] [net.minecraft.client.renderer.texture.AtlasTexture/]: Created: 256x256 textures/painting-atlas
[08Aug2019 21:50:13.012] [Client thread/INFO] [net.minecraft.client.renderer.texture.AtlasTexture/]: Created: 128x128 textures/mob_effect-atlas</br>

### Mod Version
BiomesOPlenty-1.14.4-9.0.0.211-universal</br>

**Comment by Forstride** [comment]
Can you delete the two Forge config files in the /config/ folder and see if that fixes it?

**Comment by bloodconfetti** [comment]
[crash-2019-08-09_22.05.28-server.txt](https://github.com/Glitchfiend/BiomesOPlenty/files/3488488/crash-2019-08-09_22.05.28-server.txt)

It was running good, prior to adding the mod back in, and then upon selecting my world, it crashed.

**Comment by bloodconfetti** [comment]
[crash-2019-08-09_22.05.28-server.txt](https://github.com/Glitchfiend/BiomesOPlenty/files/3488495/crash-2019-08-09_22.05.28-server.txt)
[crash-2019-08-09_22.09.51-server.txt](https://github.com/Glitchfiend/BiomesOPlenty/files/3488496/crash-2019-08-09_22.09.51-server.txt)
[crash-2019-08-09_22.12.27-server.txt](https://github.com/Glitchfiend/BiomesOPlenty/files/3488497/crash-2019-08-09_22.12.27-server.txt)

Sorry... I actually don't know if any of these apply to your mod. I don't understand coding or anything at all, really.

The first one was it crashing upon world selection after it was the only mod I added to the folder (the game had loaded previous to that.) But the second one, it crashed yet again, even after removing BOP. So I took out another mod, test ran the game, it was fine. Added BOP back into the mods folder and had the third crash upon world selection.

Sorry...
